plugins/generic/usageStats/locale/pt_BR/locale.xml

```diff
@@ -25,4 +25,17 @@
 		<li>Navegador</li>
 		</ul>
 		<p>Os dados coletados são usados apenas para propósitos de avaliação. Nenhum endereço de IP é mapeado a IDs de usuário. É tecnicamente impossível rastrear um conjunto específicos de dados a um IP específico.</p>]]></message>
+  <message key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide">&lt;h3&gt;Informações gerais sobre privacidade&lt;/h3&gt;
```

Comment:
this and the following lines: please restore HTML, your editors seems to have masked your HTML entities